js/lasson-19.js

```diff
@@ -0,0 +1,47 @@
+class ApartmentHouse {
+	numberOfApartments;
+	numberOfApartmentsPerFloor = 4;
+	numberNfFloors;
+	apartments = [];
+	constructor(numberOfApartments,numberOfApartmentResidents,
+		numberNfFloors) {
+		this.numberOfApartments = numberOfApartments;
+		this.numberNfFloors = numberNfFloors;
+		for (let index = 1; index <= this.numberOfApartments; index++) {
+			if (numberOfApartments>0||numberOfApartments<=200) {
+				this.apartments.push(new Apartment(numberOfApartmentResidents));
+			}
+		}
+	}
+}
+class Apartment {
+	numberOfApartmentResidents;
+	tenants = [];
+	constructor(numberOfApartmentResidents) {
+		this.numberOfApartmentResidents = numberOfApartmentResidents;
+		for (let index = 1; index <= this.numberOfApartmentResidents; index++) {
+			if (numberOfApartmentResidents>0) {
+				this.tenants.push(new Tenant("Alisa",18,"Woman"))
+			}
+		}
+	}
+	setTenat(name,age,gender) {
```

Comment:
```suggestion
	addTenat(name,age,gender) {
```

метод доадає мешканця, отже краще змінити назву на відповідну